### PR TITLE
C# Fundamentals : Set1 - Task4

### PR DESCRIPTION
**Pull request by tharaniannamalai**

### CODE REVIEW

File: CSharpLearning/Set1.Task4/Program.cs [hunk 1]
```diff
@@ -1,12 +1,46 @@
 ﻿using System;
 
-namespace Set1.Task4
+namespace MyApp // Note: actual namespace depends on the project name.
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task4/Program.cs]: Why comment?

File: CSharpLearning/Set1.Task4/Program.cs [hunk 2]
```diff
@@ -1,12 +1,46 @@
...
+            c = a - b;
+            Console.WriteLine("Subtract a and b values : " + c);
+
+            a = ++a;
+            b = --b;
+            Console.WriteLine("\na value : " + a);
+            Console.WriteLine("b value : " + b);
+
+            c = a * b;
+            Console.WriteLine("Muliply a and b values : " + c);
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task4/Program.cs]: Typo

File: CSharpLearning/Set1.Task4/Program.cs [hunk 3]
```diff
@@ -1,12 +1,46 @@
...
+
+            a = ++a;
+            b = --b;
+            Console.WriteLine("\na value : " + a);
+            Console.WriteLine("b value : " + b);
+
+            c = a / b;
+            Console.WriteLine("Divide a and b values : " + c);
+            Console.ReadLine();
+
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task4/Program.cs]: Remove extra line

File: CSharpLearning/Set1.Task4/Program.cs [hunk 4]
```diff
@@ -1,12 +1,46 @@
...
         }
     }
-}
+}
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task4/Program.cs]: Run code cleanup

File: CSharpLearning/Set1.Task5/Program.cs [hunk 5]
```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task5/Program.cs:1]: What is this file for?

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CSharpLearning/Set1.Task4/Program.cs (hunk 1 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task4/Program.cs]: Comment Resolved.

File: CSharpLearning/Set1.Task4/Program.cs (hunk 2 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task4/Program.cs]: Comment Resolved.

File: CSharpLearning/Set1.Task4/Program.cs (hunk 3 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task4/Program.cs]: Removed extra line.

File: CSharpLearning/Set1.Task4/Program.cs (hunk 4 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task4/Program.cs]: Comment Resolved.

File: CSharpLearning/Set1.Task5/Program.cs (hunk 5 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task5/Program.cs:1]: Comment Resolved.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**